### PR TITLE
[PTE-11901]: simplify bug report input fields for `device` and `component/platform`

### PR DESCRIPTION
**Pull request by emekafredy**
Jira link: https://syftapp.atlassian.net/browse/PTE-11901

Simplify input fields for `device` and `component/platform` for a better user experience

### CODE REVIEW

File: src/lib/support_config.ts [hunk 1]
```diff
@@ -162,15 +162,14 @@ ${submission.expected}
 
 ${product_area_submission}
 Urgent: ${submission.urgency}
-Component/Platform: ${submission.component}
+Component/Platform: ${submission.platform}
```
Comment (kieran-flex) [inline src/lib/support_config.ts:165]: Label here should be "Platform/Device"

File: src/lib/support_config.ts [hunk 2]
```diff
@@ -208,15 +207,14 @@ Submitted by: ${user.name}`;
                 `*Expected*:\n${submission.expected}\n\n` +
                 product_area_submission +
                 `*Urgent*: ${submission.urgency}\n` +
-                `*Component/Platform*: ${submission.component}\n` +
+                `*Component/Platform*: ${submission.platform}\n` +
```
Comment (kieran-flex) [inline src/lib/support_config.ts:210]: Label here should be "Platform/Device"

File: src/views/support/syft/bug.json [hunk 3]
```diff
@@ -128,28 +128,42 @@
...
           "type": "plain_text",
-          "text": "Select one or more components"
+          "text": "Select platforms/devices"
         },
         "options": [
           {
             "text": {
               "type": "plain_text",
-              "text": "BE"
+              "text": "ACP"
```
Comment (kieran-flex) [inline src/views/support/syft/bug.json:143]: Could we get the list of options in alphabetical order? I didn't mention that in the ticket, but seeing it now makes me think of it.

**Comment by emekafredy** [comment]
Updated @kieran-flex

**Pull request merged**